### ISSUE TITLE
Changelog / Version / Compatibility with MythTV

### ISSUE THREAD

**Issue by angelaschmid** [janbar/pvr.mythtv#207]
Thanks for your great Kodi Mythtv plugin, it works like a charm.

One thing what is unclear about the plugin is the compatibility with MythTV.

pvr.mythtv/changelog.txt
- contains references to the Kodi version and some Kodi API versions.
- contains references to the companion upstream library cppmyth (which doesn't have a changelog and it is unclear which MythTV version(s) is/are supported).

Normally I find references for the compatibility by reading the issues, which is very tedious.

Can you please add the compatibility level of MythTV in the changelog.txt? 
Including which MythTV versions are supported (maybe the latest 2 or 3 would do).

Since v20, the major version was changed to the Kodi release version.
Would it maybe make sense to set the minor version number to the latest MythTV version supported?

Also it would be great to find such information about the plugin on Kodi's upgrade page.

**Comment by janbar** [comment]
Yep, this information could be unclear. I try to keep the kodi pvr wiki up to date and also the addon description. Currently the addon supports MythTV versions from 0.27 to 33. I will update changelog.txt to report those informations.
